demos/interactive_face_detection_demo/face.cpp

```diff
@@ -17,17 +17,17 @@ Face::Face(size_t id, cv::Rect& location):
 }
 
 void Face::updateAge(float value) {
-    _age = (_age == -1) ? value : 0.95f * _age + 0.05f * value;
+    _age = value;
```

Comment:
What's the reason to do this?

Reply:
It was not necessary.
Returned

Reply:
So is it necessary or not? Please explain